### PR TITLE
feat(server): allow running Outline on a server with another Watchtower

### LINKED ISSUE #48: Set Watchtower to selectively watch shadowbox
I'll admit I should have read the install script a little more carefully, but while running the install, watchtower tried updating all my other containers, which ended up breaking my wordpress container :(

I would like to suggest only setting watchtower to selectively watch shadowbox.

Reading the watchtower documentation, this can be done by passing in `--label-enable` as a param on watchtower and adding `--label=com.centurylinklabs.watchtower.enable=true` on shadowbox.

More information can be found on the [watchtower repo](https://github.com/v2tec/watchtower#selectively-watching-containers)

### PR DESCRIPTION
**Pull request by AlekSi**
See https://containrrr.dev/watchtower/running-multiple-instances/

`com.centurylinklabs.watchtower.enable` and `--label-enable` are kept for backward compatibility.

Closes #48.

### CODE REVIEW

File: src/server_manager/install_scripts/install_server.sh [hunk 1]
```diff
@@ -304,6 +304,7 @@ function start_shadowbox() {
   local -ar docker_shadowbox_flags=(
     --name "${CONTAINER_NAME}" --restart always --net host
     --label 'com.centurylinklabs.watchtower.enable=true'
+    --label 'com.centurylinklabs.watchtower.scope=shadowbox'
```
Comment (fortuna) [inline src/server_manager/install_scripts/install_server.sh:307]: Will this prevent watchtower itself from being updated?

Comment (AlekSi) [reply src/server_manager/install_scripts/install_server.sh:307]: No, the label and the flag will be applied simultaneously. So old version without the label and the flag will update itself to a new version with the label and the flag.

Comment (AlekSi) [reply src/server_manager/install_scripts/install_server.sh]: Changed scope name to make it a bit more explicit. PTAL

**Comment by AlekSi** [comment]
Ping

**Comment by AlekSi** [comment]
@fortuna poke

**Review by fortuna — APPROVED** [review]

**Comment by AlekSi** [comment]
@fortuna PTAL

**Review by sbruens — APPROVED** [review]

**Pull request merged**